assets/css/main.scss

```diff
@@ -214,3 +214,50 @@ a:focus, button:focus{
 .page__content a:not(#goog-wm-sb):hover img.orcid-icon {
   box-shadow: none;
 }
+
+html {
+  scroll-behavior: smooth;
+}
+
+
+// back-to-top-button inspired by https://moderncss.dev/pure-css-smooth-scroll-back-to-top/
+// see _includes/all-events.html for the implementation
+.back-to-top-wrapper {
+  position: absolute;
+  top: 100vh;
+  right: -4rem;
```

Comment:
With ``-4`` only parts of the button are visible. I just tried for ``-2.5`` in Chrome and Safari and this might work. What do you think?

```suggestion
  right: -2.5rem;
```

Reply:
```suggestion
  right: 0;
```
just checked, that works stable when resizing the window and on mobile devices. and I think it's ok if the button overlaps with the content

![image](https://user-images.githubusercontent.com/9960249/87921094-65258500-ca7a-11ea-9fad-6c1c7d16abbb.png)